web_tree_dynamic_colored_field/demo/res_users.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+    <data>
+        <record id="view_users_tree" model="ir.ui.view">
+            <field name="model">res.users</field>
+            <field name="inherit_id" ref="base.view_users_tree" />
+            <field name="arch" type="xml">
+                <xpath expr="." position="attributes">
+                    <attribute name="color_field">lang</attribute>
```

Comment:
no, but you can verify it works as intended by inspecting the dom, it will have written the language in the color property of the style attribute. This also answers your previous question: The content of the `color_field` will stupidly be written into the style attribute. You can even use this to hack the whole thing by returning there `red; whatever_other_css_property: some_value`